content/blog/2020/09/2020-09-24-board-elections.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "2020 Jenkins Board and Officer elections. Nominations and voter registration are open!"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors: 
+- oleg_nenashev
+description: >
+  This article announces the 2020 elections and describes the next steps for contributors.
+opengraph:
+  image: TODO
+---
+
+Nominations for the 2020 elections open for one governing board position and five officer positions, namely:
+Security, Events, Release, Infrastructure and Documentation.
```

Comment:
Hyperlink to the officer descriptions for easier reading by interested individuals.
```suggestion
link:/project/team-leads/#security[Security], link:/project/team-leads/#events[Events], link:/project/team-leads/#release[Release], link:https://www.jenkins.io/project/team-leads/#infrastructure[Infrastructure], and link:https://www.jenkins.io/project/team-leads/#documentation[Documentation].
```